README.md

```diff
@@ -206,6 +206,13 @@ signIn.renderEl(
       console.log(res.user);
       res.session.setCookieAndRedirect('https://acme.com/app');
 
+      // If the widget is not configured for OIDC and the authentication type is SESSION_STEP_UP,
```

Comment:
I think it'd be nicer to call out the logic here in the code, rather than through the comment. I.e. explicitly have the:

```javascript
if (res.authTypeOrWhatever === 'SESSION_STEP_UP') {
  res.target.redirect(); 
}
else if (res.thingy === 'SESSION_SSO') {
  // do our other thing
}
```

Also, two other points:
- Were we going to use "target" or something more descriptive like "stepUp"?
- After reading through the docs, I think it'd be nicer to call it something like "finish", "complete", or "return" instead of "redirect".